docs/source/flow_setup.rst

```diff
@@ -287,6 +287,9 @@ required libraries as specified at
 <http://ray.readthedocs.io/en/latest/installation.html> and
 then follow the setup instructions.
 
+Of note is that boost is a dependency not mentioned in the additional libraries.
+Boost 1.6.5 is required, you can download the zip `here <https://dl.bintray.com/boostorg/release/1.65.0/source/boost_1_65_0.zip>`_.
```

Comment:
@kev-chien I think boost can be installed via apt-get or brew; you don't need to set it up from source?